src/resourcetiming-compression.js

```diff
@@ -1040,6 +1045,21 @@
                     }, "");
             }
 
+            if (e.workerStart && typeof e.workerStart === "number" && e.workerStart !== 0) {
+				// Has Service worker timing data that's non zero. Resource request not intercepted
+				// by Service worker always return 0 as per MDN
+				// https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/workerStart
+
+				// Lets round it and offset from startTime. We are going to round up the workerStart
+				// timing specifically. We are doing this to avoid the issue where the case of Service
+				// worker timestamps being sub-milliseconds more than startTime getting incorrectly
+				// marked as 0ms (due to round down).
+				// We feel marking such cases as 0ms, after rounding down, for workerStart would present
+				// more incorrect indication to the user. Hence the decision to round up.
+				var workerStartOffset = this.trimTiming(e.workerStart, e.startTime);
```

Comment:
No `roundUpTiming` call?

Reply:
Since we're being called for already collected and/or compressed data, rounding isn't necessary IMO since we're not protecting anything such as someones privacy with this.

Reply:
The comment you copied over mentions why we do this:

```
				// Lets round it and offset from startTime. We are going to round up the workerStart
				// timing specifically. We are doing this to avoid the issue where the case of Service
				// worker timestamps being sub-milliseconds more than startTime getting incorrectly
				// marked as 0ms (due to round down).
				// We feel marking such cases as 0ms, after rounding down, for workerStart would present
				// more incorrect indication to the user. Hence the decision to round up.
````

In Boomerang we do this first:
```
				var wsRoundedUp = roundUpTiming(e.workerStart);
```

So I'd suggest we copy over `roundUpTiming` from Boomerang and do it here too.